codefresh.yml

```diff
@@ -46,3 +46,16 @@ steps:
     when:
       branch:
         only: [ master ]
+
+  update_homebrew_formula:
+    title: "Update Homebrew formula in case version was changed"
+    image: codefresh/cli
+    fail_fast: false
+    commands:
+      - 'apk update && apk add jq'
+      - 'export PACKAGE_VERSION=$(jq -r ".version" package.json)'
+      - "echo Current version: $PACKAGE_VERSION"
```

Comment:
is there a need for `git tag`?
it is being done previously

Reply:
my mistake; just copy-paste; will fix - remove